src/Shared/Recruit.Vacancies.Client/Infrastructure/Reports/CsvBuilder.cs

```diff
@@ -43,49 +48,42 @@ private static CsvWriter GetCsvWriter(TextWriter streamWriter)
             return csv;
         }
 
-        private void WriteNoResults(CsvWriter csv, DateTime reportDate)
+        private void WriteNoResults(CsvWriter csv, IEnumerable<KeyValuePair<string, string>> headers)
         {
-            WriteTotalHeader(csv, reportDate, 0, 2);
+            WriteTotalHeader(csv, headers);
         }
 
-        private void WriteTotalHeader(CsvWriter csv, DateTime reportDate, int total, int fieldCount)
+        private void WriteTotalHeader(
+            CsvWriter csv,
+            IEnumerable<KeyValuePair<string, string>> headers)
         {
+            headers = headers ?? new KeyValuePair<string, string>[0];
```

Comment:
According to Mark's Slack message.  This needs to say `OFFICIAL` now.  Can you double-check and change.  Note that this will also update the reports for Provider.